docs/user-guide/data-structures/creating-arrays-and-datasets.ipynb

```diff
@@ -591,6 +602,23 @@
    "source": [
     "sc.compat.from_pandas(df)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### From NeXus Files\n",
+    "\n",
+    "NeXus is an HDF5-based file format data from scattering experiments.\n",
```

Comment:
Also non-scattering. Why not use how they describe it themselves? https://www.nexusformat.org/